hack/builders/README.md

```diff
@@ -0,0 +1,70 @@
+
+## Build an operator image with your custom changes
+
+```txt
+Usage: ./build_operator_image.sh [options]
+Options:
+-h, --help        show this message
+-o, --operator    operator name to build, examples: machine-config-operator, cluster-kube-controller-manager-operator
+-i, --id          id of your pull request to apply on top of the master branch
+-u, --username    registered username in quay.io
+-t, --tag         push to a custom tag in your origin release image repo, default: latest
+-d, --dockerfile  non-default Dockerfile name, default: Dockerfile
+```
+
+For instance, if you want to build a Machine Config Operator image with your custom change specified by PR [\#2606](https://github.com/openshift/machine-config-operator/pull/2606) and then push it into your personal quay.io, execute
+
+```sh
+$ ./build_operator_image.sh --username johndow --operator machine-config-operator --id 2606
+```
+
+**Note**: since Quay doesn't publish images by default, to successfully use the image, you need to make it public manually in the registry web console.
+
+## Build a release image with your custom changes
+
+```txt
+Usage: ./build_release_image.sh [options] -u <quay.io username>
+Options:
+-h, --help      show this message
+-u, --username  registered username in quay.io
+-t, --tag       push to a custom tag in your origin release image repo, default: latest
+-r, --release   openshift release version, default: 4.9
+-a, --auth      path of registry auth file, default: ./config.json
+--cccmo         custom cluster-cloud-controller-manager-operator image name, default: quay.io/openshift/origin-cluster-cloud-controller-manager-operator:4.9
+--aws-ccm       custom aws cloud-controller-manager image name, default: quay.io/openshift/origin-aws-cloud-controller-manager:4.9
+--azure-ccm     custom azure cloud-controller-manager image name, default: quay.io/openshift/origin-azure-cloud-controller-manager:4.9
+--azure-node    custom azure node manager image name, default: quay.io/openshift/origin-azure-cloud-node-manager:4.9
+--openstack-ccm custom openstack cloud-controller-manager image name, default: quay.io/openshift/origin-openstack-cloud-controller-manager:4.9
+--kapio         custom kube-apiserver-operator image name, default: current kube-apiserver-operator image from the release payload
+--kcmo          custom kube-controller-manager-operator image name, default: current kube-controller-manager-operator image from the release payload
+--mco           custom machine-config-operator image name, default: current machine-config-operator image from the release payload
+```
+
+To build an actual release image with your custom Machine Config Operator image, that was created in the previous step, execute
+
+```sh
+$ ./build_release_image.sh --username johndow --auth ~/pull-secret.txt --mco quay.io/johndow/machine-config-operator:latest
```

Comment:
here you reference the `pull-secret.txt` as the argument for the auth flag, but in the script file you reference `config.json`. i think it would be clearer to use the same name in both places, having `pull-secret.txt` here makes it very clear for me.